main.go

```diff
@@ -43,6 +43,7 @@ func main() {
 
 	defaultListenPort := fmt.Sprintf("%d", agentlocal.DefaultPMMAgentListenPort)
 	serverURLF := kingpin.Flag("server-url", "PMM Server URL in `https://username:password@pmm-server-host/` format").String()
+	kingpin.Flag("log-level", "Set logging level").EnumVar(&commands.GlobalFlags.LogLevel, "debug", "info", "warn", "error", "fatal")
```

Comment:
this flag might conflict with log-level flags that we will pass in `pmm-admin add` commands to control log level for exporters, so I suggest to not implement it here. because it makes no sense to implement it here.